extension/src/openvic-extension/classes/GUIOverlappingElementsBox.cpp

```diff
@@ -32,20 +34,14 @@ Error GUIOverlappingElementsBox::_update_child_positions() {
 	Error err = OK;
 	using enum GUI::AlignedElement::format_t;
 	switch (gui_overlapping_elements_box->get_format()) {
-		case left:
-			break;
-		case centre:
-			starting_x = (box_width - total_width) / 2;
-			break;
-		case right:
-			starting_x = box_width - total_width;
-			break;
-		default:
-			UtilityFunctions::push_error(
-				"Invalid GUIOverlappingElementsBox alignment: ",
-				static_cast<int32_t>(gui_overlapping_elements_box->get_format())
-			);
-			err = FAILED;
+	case left:	 break;
+	case centre: starting_x = (box_width - total_width) / 2; break;
+	case right:	 starting_x = box_width - total_width; break;
```

Comment:
IMO we should line break before the `break;` (and after the `case ...:` too), I believe there's a formatting setting about having multiple statements on the same line